src/components/CertificateDropZone/Views/UnverifiedView.test.js

```diff
@@ -38,58 +37,151 @@ const VALID_VERIFICATION_STATUS = {
           hash: { checksumMatch: false }
         }}
+        resetData={() => {}}
       />
-    )
-    .toJSON();
-  expect(tree).toMatchSnapshot();
-});
+    );
+    const errorContainerElm = wrapper
+      .find("DetailedErrors")
+      .dive()
+      .find("#error-tab0");
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.HASH].failureTitle
+    );
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.HASH].failureMessage
+    );
+  });
 
-it("renders correctly when the certificate is not issued", () => {
-  const tree = renderer
-    .create(
+  it("displays error if the document is not issued", () => {
+    const wrapper = shallow(
       <UnverifiedView
+        handleRenderOverwrite={() => {}}
         verificationStatus={{
           ...VALID_VERIFICATION_STATUS,
           issued: { issuedOnAll: false }
         }}
+        resetData={() => {}}
       />
-    )
-    .toJSON();
-  expect(tree).toMatchSnapshot();
-});
+    );
+    const errorContainerElm = wrapper
+      .find("DetailedErrors")
+      .dive()
+      .find("#error-tab0");
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.ISSUED].failureTitle
+    );
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.ISSUED].failureMessage
+    );
+  });
 
-it("renders correctly when the certificate is revoked", () => {
-  const tree = renderer
-    .create(
+  it("displays error if the document is revoked", () => {
+    const wrapper = shallow(
       <UnverifiedView
+        handleRenderOverwrite={() => {}}
         verificationStatus={{
           ...VALID_VERIFICATION_STATUS,
           revoked: { revokedOnAny: true }
         }}
+        resetData={() => {}}
       />
-    )
-    .toJSON();
-  expect(tree).toMatchSnapshot();
-});
+    );
+    const errorContainerElm = wrapper
+      .find("DetailedErrors")
+      .dive()
+      .find("#error-tab0");
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.REVOKED].failureTitle
+    );
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.REVOKED].failureMessage
+    );
+  });
 
-it("renders correctly when the issuers are not identified", () => {
-  const tree = renderer
-    .create(
+  it("displays error if the identity is not verified", () => {
+    const wrapper = shallow(
+      <UnverifiedView
+        handleRenderOverwrite={() => {}}
+        verificationStatus={{
+          ...VALID_VERIFICATION_STATUS,
+          identity: { identifiedOnAll: false }
+        }}
+        resetData={() => {}}
+      />
+    );
+    const errorContainerElm = wrapper
+      .find("DetailedErrors")
+      .dive()
+      .find("#error-tab0");
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.IDENTITY].failureTitle
+    );
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.IDENTITY].failureMessage
+    );
+  });
+  it("displays all errors if multiple error are there", () => {
```

Comment:
the description is a bit misleading as it implies if there are 2 errors, everything will be errored.

Suggestion => displays error in all fields when all verification fail

Reply:
fixed